src/idc_index_data/__init__.py

```diff
@@ -16,7 +16,11 @@
 
 from ._version import version as __version__
 
-__all__ = ["__version__", "IDC_INDEX_CSV_ARCHIVE_FILEPATH"]
+__all__ = [
+    "__version__",
+    "IDC_INDEX_CSV_ARCHIVE_FILEPATH",
+    "IDC_INDEX_PARQUET_FILEPATH",
```

Comment:
This also seems to be left out by accident?

Reply:
I'll comment out parquet